### PR TITLE
Me 7 https://divisionsinc.atlassian.net/browse/ME-7

### PR DESCRIPTION
**Pull request by rais-dmg**

### CODE REVIEW

**Review by venkatesh-thalluri — APPROVED** [review]

File: src/main/java/com/dmg/matchingengine/repository/TechnicianRepository.java [hunk 1]
```diff
@@ -3,8 +3,19 @@
...
 import com.dmg.matchingengine.model.entity.TechnicianEntity;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface TechnicianRepository extends JpaRepository<TechnicianEntity, String> {
+
+    String TECHNICIAN_COUNT_FOR_PROVIDER_ID_WITH_GIVEN_SERVICE_TYPE = "select count(*) from technician t  \n" +
```
Comment (anuragsikakollu) [inline src/main/java/com/dmg/matchingengine/repository/TechnicianRepository.java:13]: remove `\n` from the query and please format the query string

File: src/main/java/com/dmg/matchingengine/configuration/MatchingEngineGrpcInterceptor.java [hunk 2]
```diff
@@ -14,6 +15,13 @@
 @GRpcGlobalInterceptor
 public class MatchingEngineGrpcInterceptor implements ServerInterceptor {
 
+    // todo : test this flow when grpc contract is available
+    public final ThreadLocalUtil<Object> threadLocalUtil;
```
Comment (anuragsikakollu) [inline src/main/java/com/dmg/matchingengine/configuration/MatchingEngineGrpcInterceptor.java:19]: what is this used for?

Comment (rais-dmg) [reply src/main/java/com/dmg/matchingengine/configuration/MatchingEngineGrpcInterceptor.java:19]: to get request message data for technician count processor

File: src/main/java/com/dmg/matchingengine/MatchingEngineApplication.java [hunk 3]
```diff
@@ -4,14 +4,18 @@
...
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
 import javax.annotation.PostConstruct;
 import java.util.TimeZone;
 
 @Slf4j
 @SpringBootApplication(scanBasePackages = {"com.dmg.*"})
 @EnableConfigurationProperties
+@EnableJpaRepositories(basePackages = "com.dmg.matchingengine.repository")
```
Comment (dmittal-dmg) [inline src/main/java/com/dmg/matchingengine/MatchingEngineApplication.java:17]: Are these annotations required?

**Review by dmittal-dmg — APPROVED** [review]

**Comment by dmittal-dmg** [comment]
Also please add Jira titile or a descriptive message in MR title

File: src/main/java/com/dmg/matchingengine/MatchingEngineApplication.java (hunk 3 above)
Comment (rais-dmg) [reply src/main/java/com/dmg/matchingengine/MatchingEngineApplication.java:17]: I was getting some issues while creating beans, that's why I added this annotation.

**Pull request merged**